services/management/commands/index_search_columns.py

```diff
@@ -30,15 +29,18 @@ def get_search_column(model):
 class Command(BaseCommand):
     def handle(self, *args, **kwargs):
 
-        logger.info(
-            f"Units indexed: {Unit.objects.update(search_column=get_search_column(Unit))}"
-        )
-        logger.info(
-            f"Services indexed: {Service.objects.update(search_column=get_search_column(Service))}"
-        )
-        logger.info(
-            f"AdministrativeDivisions indexed: {AdministrativeDivision.objects.update(search_column=get_search_column(AdministrativeDivision))}"
-        )
-        logger.info(
-            f"Addresses indexed: {Address.objects.update(search_column=get_search_column(Address))}"
-        )
+        for lang in ["fi", "sv", "en"]:
+            key = "search_column_%s" % lang
+            logger.info(
+                f"{lang} Units indexed: {Unit.objects.update(**{key: get_search_column(Unit, lang)})}"
+            )
+            logger.info(
+                f"{lang} Services indexed: {Service.objects.update(**{key: get_search_column(Service, lang)})}"
+            )   
+            adm_str = f"{lang} AdministrativeDivisions indexed: " 
```

Comment:
This `adm_str` variable could be removed or simplified for code readability.

Reply:
Fixed, by removing the variable. now:
     ```
    logger.info(
                f"{lang} AdministrativeDivisions indexed: "
                f"{AdministrativeDivision.objects.update(**{key: get_search_column(AdministrativeDivision, lang)})}"
            )
```